src/main/java/andrew/BuildingGame/Game/Game.java

```diff
@@ -220,4 +225,20 @@ private void gameCleanup() {
     }
 //    BGReady.unreadyAllParticipants();
   }
+
+  private Boolean isBuildingRound() {
+    return roundNumber % 2 == 0 && isNeitherInitNorInitPromptState() && !isFinalRound(roundNumber);
+  }
+
+  private Boolean isFinalBuildingRound() {
+    return isBuildingRound() && isFinalRound(roundNumber + 2);
```

Comment:
Did you check that this works with games with both 2 and with 3 building rounds?

Off the top of my head, I'd assume that this works, but if you can double-check that would be good.

You can manually override the number of rounds + building rounds in GameVars.java (modify getNumBuildRounds() and calculateNumRounds())